share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -531,6 +531,7 @@ sxc,sexcoin,sex coin,
 sha,shacoin,sha coin,
 shade,shadecoin,shade coin,
 sdc,shadowcoin,shadow coin,
+sdc, shadowcash, shadow cash
```

Comment:
@kewde Ahh I see what you mean.  In that cause if the name has changed we can remove the `shadowncoin` triggers from the file. We also need to check if the API supports the new name already (has been updated)